tests/unit_tests/test.sh

```diff
@@ -2,14 +2,12 @@
 # This file contains unit tests for all the components.
 
 set -e
-NODE_TLS_REJECT_UNAUTHORIZED=0
-export NODE_TLS_REJECT_UNAUTHORIZED
 echo -e "\n\n==========Unit Tests=========="
 echo -e "\n==========Logger Unit Tests=========="
 cd ../../util
-npm test --silent
+TEST_TYPE='UNIT' npm test --silent
 cd ../
 
 cd load_balancer/
 echo -e "\n==========Load Balancer Unit Tests=========="
-npm test --silent
+TEST_TYPE='UNIT' npm test --silent
```

Comment:
We need invocations based on the code of all the roles. Of course, we can have one for util module.